app/lib/event_store/repository.rb

```diff
@@ -112,6 +112,7 @@ def initialize(repository = self.class.repository, event_broker = EventBroker.ne
       subscribe_for_notification(:cinstance_cancellation, Cinstances::CinstanceCancellationEvent)
       subscribe_for_notification(:cinstance_expired_trial, Cinstances::CinstanceExpiredTrialEvent)
       subscribe_for_notification(:cinstance_plan_changed, Cinstances::CinstancePlanChangedEvent)
+      subscribe_for_notification(:cinstance_plan_changed_for_buyer, Cinstances::CinstancePlanChangedForBuyerEvent)
```

Comment:
Can't we reuse `: Cinstances::CinstancePlanChangedEvent`?
Just add another method to the notifier?

Reply:
`Cinstances::CinstancePlanChangedForBuyerEvent` already inherits from `Cinstances::CinstancePlanChangedEvent`.

Reply:
Decided to merge both into one single event, one event that triggers 2 notification, like you said. Thanks for the tip 👍. https://github.com/3scale/porta/pull/709/commits/dca231b834faec27ba32ed83cbc3c6d630c71d58